addons/medical/stringtable.xml

```diff
@@ -4901,5 +4901,13 @@
             <Japanese>AI は気絶している人へ、ためらってから射撃します</Japanese>
             <Korean>기절할 경우 인공지능이 발사를 지연합니다</Korean>
         </Key>
+        <Key ID="STR_ACE_MedicalSettings_useDefaultDamageProcessing_DisplayName">
+            <English>Use default damage processing.</English>
```

Comment:
The name `default damage processing` is confusing and I am not sure by reading this what `default` means. Judging by the code, it means use Vanilla and the name for the setting should reflect that.